test/vmtests/vmtests/conftest.py

```diff
@@ -109,6 +110,18 @@ def core_legacy_azl3(request: pytest.FixtureRequest) -> Generator[Path, None, No
         raise Exception("--core-legacy-azl3 is required for test")
     yield Path(image)
 
+@pytest.fixture(scope="session")
+def output_artifacts_dir(request: pytest.FixtureRequest) -> Generator[Path, None, None]:
```

Comment:
I think "output artifacts directory" is kind of an odd name. Would "logs directory" be more descriptive for what this is?

Reply:
pending